components/reddit/sources/new-comments-on-a-subreddit/new-comments-on-a-subreddit.js

```diff
@@ -0,0 +1,125 @@
+const common = require("../common");
+const { reddit } = common.props;
+
+module.exports = {
+  ...common,
+  key: "new-comments-on-a-subreddit",
+  name: "New comments on a subreddit",
+  description:
+    "Emits an event each time a new comment is added to a subreddit.",
+  version: "0.0.1",
+  dedupe: "unique",
+  props: {
+    ...common.props,
+    subreddit: {
+      propDefinition: [reddit, "subreddit"],
+    },
+    subredditPost: {
+      type: "string",
+      label: "Post ID",
```

Comment:
You can rename to just "Post" and remove the "ID" since users are selecting posts by name from the menu.

Reply:
Yep. The prop was renamed, along with description accordingly.